reference/platform-rest-api/instance/create-instance-in-database.md

```diff
@@ -44,24 +44,25 @@ creation.
 
 **Path parameters**:
 
-| Parameter | Description |
-| --- | --- |
-| `org_slug`| Slug of the organization containing the logical database |
-| `db_name` | Name of the logical database to contain the new instance |
+| Parameter  | Description                                              |
+| ---------- | -------------------------------------------------------- |
+| `org_slug` | Slug of the organization containing the logical database |
+| `db_name`  | Name of the logical database to contain the new instance |
 
 **JSON body properties**:
 
-| Property | Required | Description |
-| --- | --- | --- |
-| `location` | yes | The location code for the location of the instance |
-| `image` | no | Type of database image to use; `latest` (default) or `canary` |
+| Property   | Required | Description                                                          |
+| ---------- | -------- | -------------------------------------------------------------------- |
+| `location` | yes      | The location code for the location of the instance                   |
+| `image`    | no       | Type of database image to use; `latest` (default) or `canary`        |
+| `group`    | no       | The name of the group to add this database to. Defaults to `default` |
```

Comment:
I don't think this is correct. 
Even though instances belong to a group, to add or delete them we use the `/groups/:group/locations` endpoints.
You probably want to make similar changes to the logical database API docs, instead of the instances API docs.

Reply:
@athoscouto I tried the following (which works with and without the `group`):

```bash
curl --location 'https://api.turso.tech/v1/databases' \
--header 'Authorization: Bearer TOKEN' \
--header 'Content-Type: application/json' \
--data '{
    "name": "db2",
    "location": "gru",
    "image": "latest",
    "group": "test"
}'
```

That's all I was attempting to document here is that databases can be assigned a group on creation. If that's wrong, happy to remove or update where it should be.

Reply:
That is correct! But the change you made on the docs do not refer to that endpoint, they refer to the `/v1/databases/:db_name/instances` endpoint.

Reply:
You're right, I updated the wrong page. It should have been [this page](https://docs.turso.tech/reference/platform-rest-api/database/create-database-in-org).